psutil/tests/test_windows.py

```diff
@@ -161,6 +161,28 @@ def test_free_phymem(self):
         self.assertAlmostEqual(
             int(w.AvailableBytes), psutil.virtual_memory().free,
             delta=TOLERANCE_SYS_MEM)
+            
+    def test_total_swapmem(self):
+        w = wmi.WMI().Win32_PerfRawData_PerfOS_Memory()[0]
+        self.assertEqual(int(w.CommitLimit) - psutil.virtual_memory().total,
+                         psutil.swap_memory().total)
+        if (psutil.swap_memory().total == 0):
+            self.assertEqual(0, psutil.swap_memory().free)
+            self.assertEqual(0, psutil.swap_memory().used)
+
+    def test_percent_swapmem(self):
+        if (psutil.swap_memory().total > 0):
+            w = wmi.WMI().Win32_PerfRawData_PerfOS_PagingFile()[0]
+            # calculate swap usage to integer percent
+            percentSwap = int(w.PercentUsage) * 100 / int(w.PercentUsage_Base)
+            # exact percent may change but should be reasonable
+            # assert within +/- 10% and between 0 and 100
+            self.assertGreaterEqual(psutil.swap_memory().percent, 0)
+            self.assertGreaterEqual(psutil.swap_memory().percent,
+                                    percentSwap - 10)
+            self.assertLessEqual(psutil.swap_memory().percent,
+                                    percentSwap + 10)
+            self.assertLessEqual(psutil.swap_memory().percent, 100)
```

Comment:
If you want to test percent with a tolerance of 10 you can rephrase this as (not tested):

```python
self.assertAlmostEqual(psutil.swap_memory().percent, percentSwap, delta=10)
```

Reply:
Thanks, didn't know that syntax.